docs/headings.md

```diff
@@ -29,19 +23,35 @@ Proper names or code snippets:
 **:thumbsdown: Incorrect** code for this rule:
 
 ```markdown
-# summer is hot
-```
 
-Proper names or code snippets:
+# The Quick Brown Fox Jumps Over The Lazy Dog
+
+# summer is hot
 
-```markdown
 # react
 
 # `String`
 
 # Init
 ```
 
+## Verb tense
+
+Use the imperative form, especially when writing the title of a task. It increases findability.
+In other words, avoid the gerundive form. Users search for "create something", not "creating something".
+
```

Comment:
Can you change that to one sentence per line?

Reply:
done :)